src/test.js

```diff
@@ -1,21 +1,54 @@
 // Import the necessary functions and modules for testing
+// Mocked function to save data to storage
+// import { expect, jest, test } from '@jest/globals';
+
+// Import the function to be tested
+import { getFromStorage, saveToStorage } from '../modules/storage.js';
```

Comment:
Great job team 🤝

- [ ] As stated by the previous reviewer, kindly ensure all your tests are passing. I can see you have done some improvements, but most of your tests are still failing.
![tests](https://github.com/sulaimonaa/to-do-list/assets/63917725/7110107a-6f97-42a6-b8b5-7412baba0f1c)